src/Tools/Generator.php

```diff
@@ -76,6 +76,36 @@ public function processRoute(Route $route, array $rulesToApply = [])
         return $parsedRoute;
     }
 
+    protected function getBodyParameters(ReflectionMethod $method, array $tags)
+    {
+        /** @var ReflectionClass $cls */
+        $cls = collect($method->getParameters())
+            ->reduce(function ($carry, $param) use ($method) {
+                if (! $param->getType() ) {
+                    return $carry;
+                }
+
+                $cls = new ReflectionClass($param->getType()->getName());
+
+                if ($cls->isSubclassOf(\Illuminate\Foundation\Http\FormRequest::class)) {
+                    return $cls;
+                }
+
+                return $carry;
+            }, null);
+
+        if ($cls) {
+            $docBlock = new DocBlock($cls->getDocComment());
+            $result = $this->getBodyParametersFromDocBlock($docBlock->getTags());
```

Comment:
Variable naming: context is good. `$result` is correct, but it can be better. How about `$bodyParametersFromFormRequestDocBlock`? Longer, but accurate.
Also, `$docBlock` in the line above this can be `$formRequestDocBlock`, just to make it clear.

People deal with a lot of cognitive overhead and make mistakes. Let's make things easier for them so mistakes are reduced.🙂

Reply:
I rewrote it to a for each loop